backend/routes.js

```diff
@@ -39,12 +40,13 @@ const configureRoutes = ({ oauthApi, prisonApi }) => {
   router.use('/api/getRoles', withErrorHandler(getRolesFactory(prisonApi).getRoles))
   router.use('/api/getUser', withErrorHandler(getUserFactory(prisonApi).getUser))
   router.use('/api/removeRole', withErrorHandler(removeRoleFactory(prisonApi).removeRole))
-  router.use('/api/addRole', withErrorHandler(addRoleFactory(prisonApi).addRole))
+  router.use('/api/addRole', withErrorHandler(addRoleFactory(oauthApi, prisonApi, logError).addRole))
   router.use('/api/contextUserRoles', withErrorHandler(contextUserRolesFactory(prisonApi).contextUserRoles))
 
   router.use(currentUser({ prisonApi, oauthApi }))
 
-  router.use('/', menuRouter({ prisonApi, logError }))
+  router.use('/', menuRouter({ logError }))
+  router.use('/manage-auth-users/:username/select-role', manageAuthUserRouter({ oauthApi, prisonApi, logError }))
```

Comment:
new select role page temporarily in new location so not used.  Will move the _Add role_ button to point to it at the same time as amending the integration tests for it